### ISSUE TITLE
PokeBuddy Enjoys The Ocean.

### ISSUE THREAD

**Issue by ToUlalume** [finndev/PokeBuddy#806]
I'm not sure if this is a cause for concern, but when using pokebuddy near any piers or body of ocean, the application seems to keep going until it is in the middle of the ocean.

I'm curious if Niantic will flag this as cheating, or as if we just enjoy going sailing for pokemon.

Thank you.

**Comment by ToUlalume** [comment]
It will also move between bodies of water going from one pier to the next...

**Comment by ToUlalume** [comment]
[P - 20:00:44] [Navigation] Walking from: 37.81276, -122.41008 to: 37.81296, -122.40988 with a distance of: 18 meters
[P - 20:00:46] [Transferring Pokemons] No Pokemons available for transfer. Trying again later...
[P - 20:00:50] [Navigation] Walking from: 37.81278, -122.41006 to: 37.81298, -122.40986 with a distance of: 18 meters

**Comment by Dj-jom2x** [comment]
does all pokemon only spawn in land?

**Comment by ToUlalume** [comment]
I don't think so... but it is just highly unlikely for people to be boarding ships just to catch pokemon in the middle of the ocean... 

or for them to get on a boat... go to the next pier... then walk... then go in the ocean... then walk onto another pier... then go in the ocean... 

thats why i'm somewhat concerned.

I found my pokebuddy way out in the ocean.

**Comment by rainyday1** [comment]
that means your IP is banned

**Comment by xXbrucyjucyXx** [comment]
just use an ip spoofer

**Comment by isoueu** [comment]
Im going to sleep. But im sure about ocean, lakes and rivers crossing giving us ban.
If have someone with time please try it in nox, with a  account that good to be banned

**Comment by r0nalxd** [comment]
What if you are on a cruise with onboard wifi? No pokemons catchable at all then? :(

**Comment by ToUlalume** [comment]
actually... the funny thing is... i'm in the ocean... or floating on the ocean near a pier... catching magikarps... and when i find myself in the middle of the ocean... i reset and i can still continue to play. SO i do not think it's an I.P. ban nor a soft ban.
